commcare/src/main/resources/webapp/js/controllers.js

```diff
@@ -426,6 +426,14 @@
             return true;
         };
 
+        $scope.validateUrlProtocol = function () {
+            if ($scope.selectedConfig.accountConfig.baseUrl.substring(0, 8) !== "https://") {
+                return false;
+            } else {
+                return true;
+            }
+        };
```

Comment:
All of this can be done in one line by returning the negation of the condition in if